docs/src/components/landing/PricingTable.js

```diff
@@ -0,0 +1,451 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { alpha } from '@mui/material/styles';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Fade from '@mui/material/Fade';
+import Paper from '@mui/material/Paper';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Divider from '@mui/material/Divider';
+import Typography from '@mui/material/Typography';
+import IconImage from 'docs/src/components/icon/IconImage';
+
+function PlanName({ planInfo, centered = false, disableDescription = false }) {
+  const { title, color, description } = planInfo;
+  return (
+    <React.Fragment>
+      {centered ? (
+        <Typography
+          variant="body2"
+          fontWeight="bold"
+          sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}
+        >
+          <IconImage name={`block-${color}`} sx={{ mr: 1 }} /> {title}
+        </Typography>
+      ) : (
+        <Typography
+          variant="body2"
+          fontWeight="bold"
+          sx={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}
+        >
+          {title} <IconImage name={`block-${color}`} />
+        </Typography>
+      )}
+      {!disableDescription ? (
+        <Typography variant="body2" color="text.secondary" sx={{ mt: 0.5, minHeight: { md: 63 } }}>
+          {description}
+        </Typography>
+      ) : null}
+    </React.Fragment>
+  );
+}
+
+PlanName.propTypes = {
+  centered: PropTypes.bool,
+  disableDescription: PropTypes.bool,
+  planInfo: PropTypes.shape({
+    color: PropTypes.string.isRequired,
+    description: PropTypes.string,
+    title: PropTypes.string.isRequired,
+  }),
+};
+
+const Cell = ({ highlighted = false, ...props }) => (
+  <Box
+    {...props}
+    sx={{
+      py: 2,
+      px: 2,
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+      justifyContent: 'center',
+      ...(highlighted && {
+        borderWidth: '0 1px 0 1px',
+        borderStyle: 'solid',
+        borderColor: (theme) => (theme.palette.mode === 'dark' ? 'primaryDark.700' : 'grey.100'),
+        bgcolor: (theme) =>
+          theme.palette.mode === 'dark'
+            ? alpha(theme.palette.primaryDark[900], 0.5)
+            : alpha(theme.palette.grey[50], 0.5),
+      }),
+      ...props.sx,
+    }}
+  />
+);
+
+Cell.propTypes = {
+  highlighted: PropTypes.bool,
+  sx: PropTypes.object,
+};
+
+const RowHead = ({ children, startIcon, ...props }) => (
```

Comment:
when `props` does not include all the props but just the remaining ones i usually call them `rest` just so that it's a bit more obvious - it's just a tip and the way that i do it in case it's helpful